loader/include/Geode/utils/web.hpp

```diff
@@ -60,6 +60,52 @@ namespace geode::utils::web {
         bool certVerification = true; // Enable HTTPS certificate verification
     };
 
+    class GEODE_DLL MultipartForm final {
+    private:
+        class Impl;
+
+        std::shared_ptr<Impl> m_impl;
+
+    public:
+        MultipartForm();
+
+        MultipartForm& param(std::string_view name, std::string_view value);
+        template <std::integral T>
+        MultipartForm& param(std::string_view name, T value) {
+            return this->param(name, std::to_string(value));
+        }
+
+        MultipartForm& file(std::string_view name, ByteVector const& data, std::string_view filename, std::string_view mime = "application/octet-stream");

```

Comment:
`ByteVector const& data` since this cant move the vector, might as well use a `std::span<u8 const>`